### PR TITLE
add tests for emotions estimations

### PR DESCRIPTION
**Pull request by gluhar2006**

### CODE REVIEW

**Comment by matemax** [comment]
+ test with no warped image

**Comment by gluhar2006** [comment]
> * test with no warped image

not possible

File: tests/test_estimate_emotions.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        self.validate_emotion_dict(emotionDetection)
+        if predominantEmotion is not None:
+            assert emotionDetection["predominant_emotion"], predominantEmotion
+
+    @staticmethod
+    def validate_emotion_dict(receivedDict: dict):
+        """
+        Validate emotion dict
+        """
+        assert sorted(["predominant_emotion", "estimations"]), sorted(receivedDict.keys())
```
Comment (matemax) [inline tests/test_estimate_emotions.py]: .

File: tests/test_estimate_emotions.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        assert sorted(ALL_EMOTIONS), receivedDict["estimations"]
```
Comment (matemax) [inline tests/test_estimate_emotions.py]: ...

**Review by matemax — APPROVED** [review]

File: tests/test_estimate_emotions.py (hunk 1 above)
Comment (gluhar2006) [reply tests/test_estimate_emotions.py]: updated

File: tests/resources.py [hunk 3]
```diff
@@ -11,3 +11,6 @@ def getPathToImage(filename: str) -> str:
 MANY_FACES = getPathToImage("many_faces.jpg")
 NO_FACES = getPathToImage("kand.jpg")
 SMALL_IMAGE = getPathToImage("small_image.jpg")
+
+ALL_EMOTIONS = ["anger", "disgust", "fear", "happiness", "sadness", "surprise", "neutral"]
```
Comment (matemax) [inline tests/resources.py]: from enum

**Review by matemax — APPROVED** [review]

File: tests/resources.py (hunk 3 above)
Comment (gluhar2006) [reply tests/resources.py]: updated

**Pull request merged**